### ISSUE TITLE
Delivery Choices API or not for Address Lookup

### ISSUE THREAD

**Issue by ProxiBlue** [fontis/fontis_australia#60]
Your docs and comments in admin setup states that user will require a DCE Key for performing address validation lookup. which is found from the Delivery Choices api.

> You will need to fill in the Australia Post Delivery Choices section to use the Australia Post address validation service.</comment>

however, speaking with austpost, they claim there is a seperate API for address lookups, and is not linked to Delivery Choices for address validation.

so, considering this: Is the Address Lookup/Validation of this module still functional, as it seems API's were changed at Austpost

**Comment by ProxiBlue** [comment]
Let me rephrase the above. Basically we cannot seem to source an API key for the given address lookup, as Australia Post tells us that there is no address validtion service on Delivery Choices.

This can be verified at the API docs:
https://developers.auspost.com.au/apis/dce/reference

Your module uses an endpoint of : ValidateAddress.json - which is not listed.

Is this undocumented endpoint going to to be in use going forward by austpost?

**Comment by digitalpianism** [comment]
@ProxiBlue did you ever find out about this ? I find weird that this is not listed in the API reference.

**Comment by ProxiBlue** [comment]
@digitalpianism As far as my conversations with AustPost went, the discontinued that API, or it was never actually released. Information to pin that down was scetchy.

I eventually rolled a local address validation that uses the local db of existing postcode / state / suburb combinations

https://github.com/ProxiBlue/fontis_australia/tree/address_validation_using_local_db

**Comment by Puru2016** [comment]
After installed the latest version(3.4.1) I am getting Fatal error:  Class 'Auspost\Common\Auspost' not found in /var/www/html/zapsales/app/code/community/Fontis/Australia/Helper/Address/DeliveryChoices.php on line 17

Please help me to fix this issue OR suggest any fix.
